components/PageControl/examples/PageControlTypicalUseExample.m

```diff
@@ -149,4 +165,16 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+- (CGFloat)xOffsetForPage:(NSInteger)page
+            numberOfPages:(NSInteger)numberOfPages
+                    width:(CGFloat)width {
+  CGFloat offsetMultiplier = [self isRTL] ? numberOfPages - page - 1 : page;
```

Comment:
The `numberOfPages - page - 1 : page` logic is being duplicated above and is still a good candidate for being a method.

To clarify, the intent with this feedback is to reduce duplication of relatively complex logic by consolidating it into a single method.

Reply:
@jverkoey, what if we have one method that returns offsetMultiplier and another that returns xOffset and that calls the former? How's that sound?

Reply:
@jverkoey, with ffd9ea9 the example now has one ternary -- will this suffice?